scripts/arr.js

```diff
@@ -0,0 +1,60 @@
+function sumMultiply(array) {
+  if (!Array.isArray(array) || array.lenth !==4) {
+    return 'Array is expected to have 4 numbers';
+  }
+
+  return array[0] * array[1] + array[2] * array [3];
+}
+function runSumMultiply() {
+  const userArray = prompt('Enter array:', '[2, 5, 3, 9]') || [];
+
+  const result = sumMultiply(userArray);
```

Comment:
```suggestion
  const userNumbers = prompt('Enter 4 numbers divided by space:', '2 5 3 9') || '0';
  
  const array = userNumbers.split(' ')

  const result = sumMultiply(array);
```

Reply:
So we adjusted the user input. Add the split method to split a string into substrings. When the user enters numbers with spaces, the code works
If the user enters the numbers in their own way, it will not be possible to calculate array. 
The code is working!!!

Reply:
The reson is: `prompt` function returns a string, not an object. That's why we have to receive an input easy enough to transform into an array.

Reply:
Yes)) Thank you!!